claim/views.py

```diff
@@ -36,17 +51,31 @@ def get_claims(request, polygon_id):
 
 @login_required
 def add_claim(request):
+    if not request.POST.get('g-recaptcha-response', False):
```

Comment:
Let's make 3 types of users: anonymous, accounted and de-anonymized. Thus, anonymous will get capcha in 100% of claims, accounted user - 20% and de-anonymized 0%. Capcha in this model is spam protection, accounts+capcha is idiot protection. Also such differentiation should stimulate users to create accounts and\or intensify civil activity.